@here/harp-mapview/lib/text/TextStyleCache.ts

```diff
@@ -373,7 +370,7 @@ export class TextStyleCache {
                     backgroundOpacity = backgroundOpacity * alpha;
                     hexBgColor = ColorUtils.removeAlphaFromHex(hexBgColor);
                 }
-                this.m_colorMap.set(cacheId + "_bg", ColorCache.instance.getColor(hexBgColor));
```

Comment:
was there some reason for having the background colors separately cached?

Reply:
I don't know, this code looks like leftover as current this method is merge of duplicated functions from PoiRenderer/TileGeometryCreator.
Look, that this entries in `m_colorMap` always rewritten, so there is no caching at all. Effectively, it was obfuscated local variable. That's why i changed it to local variables.

Reply:
Ok